RELEASES.md

```diff
@@ -1,3 +1,163 @@
+Version 1.27.0 (2018-06-21)
+==========================
+
+Language
+--------
+- [Removed 'proc' from the reserved keywords list.][49699] This allows `proc` to
+  be used as an identifer.
+- [The dyn syntax is now available.][49968] This syntax is equivalent to the
+  bare `Trait` syntax, and should make it clearer when being used in tandem with
+  `impl Trait`. Since it is equivalent to follow syntax has the same behaviour.
+  `&Trait == &dyn Trait`, `&mut Trait == &mut dyn Trait`, and
+  `Box<Trait> == Box<dyn Trait>`.
+- [Attributes on generic parameters such as types and lifetimes are
+  now stable.][48851] e.g.
+  `fn foo<#[lifetime_attr] 'a, #[type_attr] T: 'a>() {}`
+- [The `#[must_use]` attribute can now also be used on functions as well as
+  types.][48925] It provides a lint that by default warns users when the
+  return type is not being used.
+
+Compiler
+--------
+- [Added the `armv5te-unknown-linux-musl` target.][50423]
+
+Libraries
+---------
+- [SIMD(Single Instruction Multiple Data) on x86/x86_64 is now stable.][49664]
+  This includes [`arch::x86`] & [`arch::x86_64`] modules which contain
+  SIMD intrinsics, a new macro called `is_x86_feature_detected!`, the
+  `#[target_feature(enable="")]` attribute, and adding `target_feature = ""` to
+  the `cfg` attribute.
+- [A lot of methods for `[u8]`, `f32`, and `f64` previously only available in
+  std are now available in core.][49896]
+- [`memchr::repeat_byte` is now optimised to have a reduced code
+  size footprint.][50398] It will also be slightly faster on 64bit platforms.
```

Comment:
I'm not sure this item worth putting into the release note. The original PR doesn't have the `relnote` label either.

Reply:
Release notes have never been exclusively pull requests with the relnotes label. I always like including optimisations as it is something a lot of people are interested in. If you feel strongly about its inclusion I'm fine with removing it.

Reply:
I do feel quite strongly because `memchr::repeat_byte` isn't a public API. 

We could say public APIs like `<str>::{contains, find, rfind, matches, split, replace}` with a `char` pattern becomes a bit faster.

Reply:
I completely forgot that memchr is not in the public API. 😅